WeaveApp/webpack.prod.js

```diff
@@ -0,0 +1,23 @@
+const merge = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const Visualizer = require('webpack-visualizer-plugin');
+const common = require('./webpack.common.js');
+const webpack = require('webpack');
+
+module.exports = merge(common, {
+    devtool: 'source-map',
+    plugins: [
+        new UglifyJSPlugin({
```

Comment:
This plugin uglify's and minify's the compiled code. It also creates a source map.